firmware/src/rdm_responder.h

```diff
@@ -18,13 +18,24 @@
  */
 
 /**
- * @defgroup rdm_responder RDM Responder
- * @brief The RDM Responder Subsystem.
- *
- * @addtogroup rdm_responder
+ * @addtogroup rdm
  * @{
  * @file rdm_responder.h
- * @brief The RDM Responder Subsystem.
+ * @brief The base RDM Responder.
+ *
+ * The base RDM Responder provides the building blocks for implementing
+ * responder models. You can think of it as a base class if we were using C++.
+ *
+ * It consists of a couple of parts:
+ *  - the global g_responder object, which holds basic state like mute,
+ *    identify etc.
+ *  - The PID dispatching mechanism, where we specifiy a table of function
+ *    pointers and then call RDMResponder_DispatchPID().
+ *  - Functions for various common PIDs.
+ *
+ * When implementing a model, you can reference the PID functions in the
+ * dispatch table, or point to your own functions that (optionally) wrap the
+ * PID functions.
```

Comment:
So you can override the default identify device for example, to produce a faulty responder?